module/documents/actor/actor.mjs

```diff
@@ -1639,7 +1639,7 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
    * @returns {Promise<D20Roll[]|null>}                        A Promise which resolves to the created Roll instances.
    */
   async rollSavingThrow(config={}, dialog={}, message={}) {
-    const abilityLabel = CONFIG.DND5E.abilities[config.ability]?.label ?? "";
+    const abilityLabel = config.isConcentration ? game.i18n.localize("DND5E.Concentration") : CONFIG.DND5E.abilities[config.ability]?.label ?? "";
```

Comment:
Move this into the `rollConcentration` method, this method really shouldn't know about the existence of concentration saves as a concept.